core/variable_model.js

```diff
@@ -87,6 +94,21 @@ Blockly.VariableModel.prototype.getId = function() {
 };
 
 /**
+ * Disable the variable.
+ */
+Blockly.VariableModel.prototype.disable = function() {
+  this.disabled = true;
+  Blockly.Events.fire(new Blockly.Events.VarDisable(this));
+};
+
+/**
+ * Enable the variable.
+ */
+Blockly.VariableModel.prototype.enable = function() {
+  this.disabled = false;
```

Comment:
This is a good point. I found a bug from that thank you! Added a an enable event.